modules/nodes-nodes-swap-memory.adoc

```diff
@@ -10,7 +10,7 @@
 :FeatureName: Enabling swap memory use on nodes
 include::snippets/technology-preview.adoc[]
 
-You can enable swap memory use for {product-title} workloads on a per-node basis.
+Except for container-native virtualization (CNV) users or use cases, enabling swap memory is not available to {product-title} users.
```

Comment:
```suggestion
Enabling swap memory is only available for container-native virtualization (CNV) users or use cases.
```

Reply:
Hello @brendan-daly-red-hat,
   Reading through the original JIRA ticket, I think this suggestion would make sense. Also, I wonder if the placement should be in some sort of call out?  I realize the updated content is currently in between two call outs already, but at first glance it seems like it is an important enough detail to be made more prominent.